db/schema.rb

```diff
@@ -39,40 +35,32 @@
     t.index ["phone_number"], name: "index_users_on_phone_number", unique: true
   end
 
-  create_table "voters", primary_key: "sos_id", id: :serial, force: :cascade do |t|
+  create_table "voters", primary_key: "reach_id", id: :string, force: :cascade do |t|
+    t.string "sos_id"
     t.string "last_name"
     t.string "first_name"
     t.string "middle_name"
+    t.string "suffix"
+    t.string "prefix"
     t.integer "age"
     t.string "gender"
+    t.string "email"
+    t.string "phone_country_code"
     t.string "primary_phone_number"
     t.string "voting_street_address"
+    t.string "voting_state"
     t.string "voting_city"
     t.string "voting_zip"
-    t.string "support_score"
-    t.string "vote_plan"
+    t.string "voter_registration"
+    t.json "voting_history"
     t.string "voting_status"
-    t.boolean "voted_general"
-    t.float "gotv_score"
+    t.integer "voting_address_id"
```

Comment:
whats this field for?

Reply:
Will be null for us most likely. Ossoff used it to match to some DB of polling locations backed by some public API. Since it was a public API rather than some internal things, I figure we might as well keep the field around in case we want it later